Dockerfile

```diff
@@ -6,14 +6,14 @@ RUN apt-get update && apt-get install -y \
     libssl-dev \
     logrotate \
     sudo \
-&& rm -rf /var/lib/apt/lists/*
+    && rm -rf /var/lib/apt/lists/*
 
 ENV VERNEMQ_VERSION 0.15.3
 
 ADD https://bintray.com/artifact/download/erlio/vernemq/deb/jessie/vernemq_$VERNEMQ_VERSION-1_amd64.deb /tmp/vernemq.deb
+ADD https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh /usr/sbin/wait-for-it.sh
```

Comment:
yeah, that'd do it too, though wait-for-it.sh allows you to if you wanted to wait for all the listeners to come up. i don't know if thats necessarily a thing that you want; you could of course chain to get the same thing.
My current use case is https://github.com/erlio/docker-vernemq/pull/11/files#diff-e43a532580d1e894a963b6cbdf8c2c91R72